util/auto_skip_compressor.cc

```diff
@@ -62,14 +62,18 @@ Status AutoSkipCompressorWrapper::CompressBlock(
       "AutoSkipCompressorWrapper::CompressBlock::exploitOrExplore",
       &exploration);
   if (exploration) {
+    auto autoskip_wa = static_cast<AutoSkipWorkingArea*>(wa->get());
```

Comment:
We still have three ugly `static_cast<AutoSkipWorkingArea*>` where I believe one should be sufficient.